### PR TITLE
whitelist + reorg round 1

### LINKED ISSUE #1555: Document Registry Scope (System vs Project Registries) and Whitelist enforcement
Per https://github.com/vmware/vic-planning/issues/12 (Epic)

### PR DESCRIPTION
**Pull request by alextopuzov**
Fixes #1555 

+ first round of topic reorg with more containers
+ three dot fix
+ UI changes fixes

### CODE REVIEW

File: docs/user_doc/vic_cloud_admin/add_repos_in_portal.md [hunk 1]
```diff
@@ -1,20 +1,6 @@
...
-**V1 over HTTP (unsecured, plain HTTP registry)** | You can freely search this kind of registry, but you must manually configure each Docker host with the `--insecure-registry` flag to provision containers based on images from insecure registries. You must restart the Docker daemon after setting the property. You cannot use HTTP connections with vSphere Integrated Containers Registry instances.
-**V1 over HTTPS** | Use behind a reverse proxy, such as NGINX. The standard implementation is available through open source at https://github.com/docker/docker-registry.
-**V2 over HTTPS** | The standard implementation is open sourced at https://github.com/docker/distribution.
-**V2 over HTTPS with basic authentication** | The standard implementation is open sourced at https://github.com/docker/distribution.
-**V2 over HTTPS with authentication through a central service** | You can run a Docker registry in standalone mode, in which there are no authorization checks.
+You can add multiple global registries that are available to all users of the management portal. Global registries that are allowed by the cloud admin cannot be disabled or removed by other users.
```
Comment (stuclem) [inline docs/user_doc/vic_cloud_admin/add_repos_in_portal.md]: "Global registries that are **added** by the cloud admin...."

File: docs/user_doc/vic_cloud_admin/working_with_registries.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Working with Registries #
+
+Use registries to store and distribute images. You can add multiple registries, in addition to the integrated vSphere Integrated Containers Registry to gain access to both public and private images. You can enable and disable the registries that you added. You can add global registries, that are visible to all projects, as well as project registries, that are available only to the project to which they are added. All users can only search and provision images and templates from registries that are available to their projects. When you disable a registry, searching for templates and images in that registry is also disabled.
```
Comment (stuclem) [inline docs/user_doc/vic_cloud_admin/working_with_registries.md]: "You use registries to store....."

Comment (stuclem) [reply docs/user_doc/vic_cloud_admin/working_with_registries.md]: "... as well as project registries..." ==> internal registries?

File: docs/user_doc/vic_cloud_admin/working_with_registries.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Working with Registries #
+
+Use registries to store and distribute images. You can add multiple registries, in addition to the integrated vSphere Integrated Containers Registry to gain access to both public and private images. You can enable and disable the registries that you added. You can add global registries, that are visible to all projects, as well as project registries, that are available only to the project to which they are added. All users can only search and provision images and templates from registries that are available to their projects. When you disable a registry, searching for templates and images in that registry is also disabled. 
+
+Even if you disable the default https://registry.hub.docker.com registry, you can still see the popular templates under **Library** > **Repositories**. To customize your popular templates, see [Customize the Popular Templates list](https://github.com/vmware/admiral/wiki/Configuration-guide#customize-the-popular-templates-list) documentation.
```
Comment (stuclem) [inline docs/user_doc/vic_cloud_admin/working_with_registries.md]: Can we recreate the popular templates information into our docs?

File: docs/user_doc/vic_cloud_admin/working_with_registries.md (hunk 2 above)
Comment (alextopuzov) [reply docs/user_doc/vic_cloud_admin/working_with_registries.md]: internal repositories / global registries / project registries

**Pull request merged**